forte/models/da_rl/magic_model.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+A model that copies the parameter states of a pytorch nn module
+and performs parameter updates locally.
+"""
+
+import copy
+from typing import Optional, Tuple, Dict
+import operator
+import torch
+import torch.nn as nn
+
+
+__all__ = [
+    "MetaModule"
+]
+
+
+class MetaModule(nn.Module):
+    # pylint: disable=line-too-long
```

Comment:
Let's not disable this thing.

Reply:
I disabled this because I have some URLs in the comments and they should be in one single line? I made sure that all the other lines are <80 char long.